src/main/java/emissary/core/IBaseDataObjectXmlCodecs.java

```diff
@@ -431,7 +431,7 @@ public ElementEncoders(
      * object.
      */
     public static final ElementDecoder DEFAULT_STRING_OBJECT_DECODER = (elements, ibdo, ibdoMethodName) -> {
-        final Method method = getIbdoMethod(ibdoMethodName, String.class, Object.class);
+        final Method method = getIbdoMethod(ibdoMethodName, String.class, CharSequence.class);
```

Comment:
Hmm..  for parameters, we're encoding `String` _(param name)_ and `Collection<Object>` _(value)_, but now decoding the Objects explicitly as CharSequences, explicitly breaking the `StringObjectEncoder`/`StringObjectDecoder` symmetry. . It may have been implicitly broken already though, since the base StringObjectEncoder calls `toString()` when writing the XML element value.